kimchi/src/circuits/polynomials/range_check/gate.rs

```diff
@@ -18,17 +18,42 @@ use crate::{
         expr::{self, l0_1, Environment, E},
         gate::{CircuitGate, GateType},
         polynomial::COLUMNS,
-        wires::GateWires,
+        wires::{GateWires, Wire},
     },
 };
 
 use super::{RangeCheck0, RangeCheck1};
 
+// Connect the pair of cells of the witness specified by the cell1 and cell2 parameters
+// cell1 --> cell2 && cell2 --> cell1
```

Comment:
Can you add a warning to this comment, saying that this will split the permutation if the two elements are transitively already part of the same permutation

Reply:
👍🏼